pkg/api/auth_middleware.go

```diff
@@ -62,7 +61,8 @@ func AuthMiddleware(logger logging.Logger, swagger *openapi3.Swagger, authentica
 				return
 			}
 			if user != nil {
-				r = r.WithContext(auth.WithUser(r.Context(), user))
+				ctx := logging.AddFields(r.Context(), logging.Fields{logging.UserFieldKey: user.Username})
+				r = r.WithContext(auth.WithUser(ctx, user))
```

Comment:
The S3 gateway also has [this line](https://github.com/treeverse/lakeFS/blob/0dec04af1aaa97d95a8441cee01fec966477c4ca/pkg/gateway/middleware.go#L67).  Do we need to add it here?  (Perhaps delete it over there?)

Reply:
I don't think the mentioned line should be here since it represents additional context related to the sig s3 gw operations.